### ISSUE TITLE
Bug in double dummy estimation

### ISSUE THREAD

**Issue by ThePokerDude** [EdwardPiwowar/BBA#834]
Hand from https://github.com/EdwardPiwowar/BBA/issues/646
002E0A054AE6DA1BFFE7075198D5
3nt -7 must be a very special declarer play ;)
2d -6 looks also pretty impossible

![image](https://github.com/user-attachments/assets/d0055493-859d-451a-acd3-4539255a6f80)

**Comment by ThePokerDude** [comment]
Repasting the board in BBA gives a completely different result
![image](https://github.com/user-attachments/assets/595b46dd-daba-40e6-9b94-30107c7676ba)

**Comment by EdwardPiwowar** [comment]
I think DD is correct but the tricks displayed are for the previous hand.
I think DD is correct but the tricks for the previous hand are displayed.
I don't know how to repeat this situation. Do you remember how you did it?

**Comment by ThePokerDude** [comment]
Not exactly, I could not reproduce it. I know it the estimation is from 
1A183AFAED2AF15683C1436C2D25
because this was the previous board.
I'll close it. Next time I'll run into something like this I will pay attention to what I did before.

**Comment by EdwardPiwowar** [comment]
The reason is that the next hand was introduced too quickly. It should be possible to improve.
